code_benches/run.py

```diff
@@ -76,13 +96,13 @@ def __str__(self):
                               "-Wno-shift-negative-value"]),
     Program("custom_matrix_multiply", [], 2 ** 14),
     Program("custom_memcmp", [], 2 ** 14),
-    Program("custom_sqlite", [], 2 ** 15),
+    Program("custom_sqlite", [], 2 ** 15, custom_arguments=["-DSQLITE_MUTEX_NOOP", "-ldl"]),
```

Comment:
Note to self: need to test on OSX

Reply:
That is something I can't really do so I'll leave it up to you :)